### PR TITLE
Add Paging for Audit Log

### PR DESCRIPTION
**Pull request by EdiWang**

### CODE REVIEW

File: src/Moonglade.Auditing/MoongladeAudit.cs [hunk 1]
```diff
@@ -96,21 +95,32 @@ public async Task<Response> AddAuditEntry(EventType eventType, EventId eventId,
                             AND (@EventId IS NULL OR al.EventId = @EventId)
                             ORDER BY al.EventTimeUtc DESC
                             OFFSET @Skip ROWS
-                            FETCH NEXT @Take ROWS ONLY";
+                            FETCH NEXT @Take ROWS ONLY
 
-                var entries = await conn.QueryAsync<AuditEntry>(sql, new
+                            SELECT COUNT(al.Id)
```
Comment (Anduin2017) [inline src/Moonglade.Auditing/MoongladeAudit.cs]: Seems you are using the second SQL to get total counts.

**Comment by Anduin2017** [comment]
But, how can I disable the audit feature to prevent tracking and protect the privacy and get better performance?

**Comment by EdiWang** [comment]
> But, how can I disable the audit feature to prevent tracking and protect the privacy and get better performance?

https://github.com/EdiWang/Moonglade/commit/f373de21ac930d4992f23266ab6c95a77c8db099

File: src/Moonglade.Auditing/MoongladeAudit.cs (hunk 1 above)
Comment (EdiWang) [reply src/Moonglade.Auditing/MoongladeAudit.cs]: This is right, it's QueryMutilple feature to enhance performance

**Comment by Anduin2017** [comment]
Cool!

**Pull request merged**